include/fastdds/rtps/common/GuidPrefix_t.hpp

```diff
@@ -93,11 +94,12 @@ inline std::ostream& operator <<(
         const GuidPrefix_t& guiP)
 {
     output << std::hex;
+    output.fill('0');
     for (uint8_t i = 0; i < 11; ++i)
     {
-        output << (int)guiP.value[i] << ".";
+        output << std::setw(2) << (int)guiP.value[i] << ".";
     }
-    output << (int)guiP.value[11];
+    output << std::setw(2) << (int)guiP.value[11];
```

Comment:
This will permanently modify the fill character of the output. The current fill character should be restored at the end.
```suggestion
    chat old_fill = output.fill('0');
    for (uint8_t i = 0; i < 11; ++i)
    {
        output << std::setw(2) << (int)guiP.value[i] << ".";
    }
    output << std::setw(2) << (int)guiP.value[11];
    output.fill(old_fill);
```

Reply:
Additionally, these tests are consistently failing on all platforms:
```
BlackboxTests_RTPS.PersistenceGuid.CheckPrevalenceBetweenManualAndPropertyConfiguration.Transport
BlackboxTests_RTPS.PersistenceGuid.CheckPrevalenceBetweenManualAndPropertyConfiguration.Intraprocess
BlackboxTests_DDS_PIM.PubSubBasic.ReceivedPropertiesDataWithinSizeLimit.Transport
BlackboxTests_DDS_PIM.PubSubBasic.ReceivedPropertiesDataWithinSizeLimit.Intraprocess
BlackboxTests_DDS_PIM.PubSubBasic.ReceivedPropertiesDataWithinSizeLimit.Datasharing
```

Reply:
I've address the fill on daca853. I'll address the failing tests separately